### ISSUE TITLE
Using list accessibility features

### ISSUE THREAD

**Issue by paderick** [AbdulRahmanAlHamali/flutter_typeahead#416]
Function `createSuggestionsWidget` should be improved for accessibility, e.g. by using `semanticChildCount`.

```dart
Widget createSuggestionsWidget() {
    Widget child = ListView(
      padding: EdgeInsets.zero,
      primary: false,
      shrinkWrap: true,
      controller: _scrollController,
      reverse: widget.suggestionsBox!.direction == AxisDirection.down
          ? false
          : true, // reverses the list to start at the bottom
      semanticChildCount: this._suggestions.length,
      addSemanticIndexes: true,
      children: this._suggestions!.map((T suggestion) {
        return InkWell(
          child: widget.itemBuilder!(context, suggestion),
          onTap: () {
            widget.onSuggestionSelected!(suggestion);
          },
        );
      }).toList(),
    );

    if (widget.decoration!.hasScrollbar) {
      child = Scrollbar(
        controller: _scrollController,
        child: child,
      );
    }

    return child;
  }
```

**Comment by sjmcdowall** [comment]
Submit a PR with documentation and a test (and example) ??  This is a community driven component .. Cheers

**Comment by clragon** [comment]
`addSemanticIndexes` is true by default in ListView.
`semanticChildCount` defaults to the `itemCount`.
There is no need to specify these manually.
